docker/AIDockerImage/requirements.txt

```diff
@@ -0,0 +1,51 @@
+aiohappyeyeballs==2.6.1 ; python_version >= "3.12" and python_version < "4.0"
```

Comment:
no need to add requirement.txt file, please use pyproject.toml to set the dependencies